src/core/Link/LinkList/LinkList.md

```diff
@@ -0,0 +1,46 @@
+A styled list for providing a list of links.
+
+- Use a descriptive heading or a label with the list.
+- Link heading or label to the list via the `ariaDescribedBy` prop.
+- Use the right heading level semantically and use the Heading component's `as` prop for changing the styling if needed.
+- Wrap each child in `LinkListItem` component for the correct styling.
+- `LinkListItem` supports `Link`, `ExternalLink` and `RouterLink` as its children.
+
+```js
+import {
+  Link,
+  ExternalLink,
+  LinkListItem,
+  RouterLink,
+  LinkList,
+  Heading
+} from 'suomifi-ui-components';
+
+<div>
+  <Heading variant="h5" as="h3" id="heading">
+    Useful resources
+  </Heading>
+  <LinkList aria-describedby="heading">
```

Comment:
This should be `ariaDescribedBy` since it's a mandatory prop in TS. Styleguidist let's this slide...